src/components/ModelizerDrawView.vue

```diff
@@ -131,21 +123,10 @@ async function drawComponents() {
 }
 
 /**
- * Display an error message to the user.
- */
-function notifyError() {
-  Notify.create({
-    type: 'negative',
-    message: t('errors.templates.getData'),
-    html: true,
-  });
-}
-
-/**
- * Update plugins array and related component templates array.
+ * Update plugin and related component templates array.
  * @return {Promise<void>} Promise with nothing on success otherwise an error.
  */
-async function updatePluginsAndTemplates() {
+async function InitView() {
```

Comment:
Rename `drawComponents` to `initComponents`
and when you `drawComponents();` in `initView` replace it by:
```
initComponents();
data.plugin.draw('root');
```

Reply:
`InitView` -> `initView`

Reply:
update the doc

Reply:
Move new method `initComponents` in `PluginManager`